### DISCUSSION TITLE
Published: Predictive Performance of Physiology-Based Pharmacokinetic Dose Estimates for Pediatric Trials

### DISCUSSION THREAD

**Discussion by Yuri05** [Open-Systems-Pharmacology/Forum#928] [Announcement]
*Predictive Performance of Physiology-Based Pharmacokinetic Dose Estimates for Pediatric Trials: Evaluation With 10 Bayer Small-Molecule Compounds in Children*
https://github.com/Open-Systems-Pharmacology/OSP-based-publications-and-content/issues/310

_Development and guidance of dosing schemes in children have been supported by physiology-based pharmacokinetic (PBPK) modeling for many years. PBPK models are built on a generic basis, where compound- and system-specific parameters are separated and can be exchanged, allowing the translation of these models from adults to children by accounting for physiological differences. Owing to these features, PBPK modeling is a valuable approach to support clinical decision making for dosing in children. In this analysis, we evaluate pediatric PBPK models for 10 small-molecule compounds that were applied to support clinical decision processes at Bayer for their predictive power in different age groups. Ratios of PBPK-predicted to observed PK parameters for the evaluated drugs in different pediatric age groups were estimated. Predictive performance was analyzed on the basis of a 2-fold error range and the bioequivalence range (ie, 0.8 ≤ predicted/observed ≤ 1.25). For all 10 compounds, all predicted-to-observed PK ratios were within a 2-fold error range (n = 27), with two-thirds of the ratios within the bioequivalence range (n = 18). The findings demonstrate that the pharmacokinetics of these compounds was successfully and adequately predicted in different pediatric age groups. This illustrates the applicability of PBPK for guiding dosing schemes in the pediatric population._

**Comment by drmansoor** [comment]
I hope you are keeping well,our research team has been involved in the silico studies using pk sim since the beginning of the system.We are first who got the licence of pk sim  in Pakistan. My colleague prof.Dr.Tasneem Ahmed along with my student name Sharib also joined Bayer in Germany to attend the training seminar. We also published a few articles and currently our group started a project related to pk sim. We will appreciate it if you supervise and guide in this regard. 
kindly, provide me your email address for further discussion .
Thanks 
Best  Regards
Prof.Dr.Mansoor Ahmed.
department: pharmaceutical chemistry , university of Karachi Pakistan
